requirements/torch.txt

```diff
@@ -3,3 +3,4 @@ torchvision
 # Note: pytorch_lightning>=1.5 is required for CLI
 # https://github.com/lightly-ai/lightly/issues/912
 pytorch_lightning>=1.0.4
+timm
```

Comment:
Why can't we make it an optional dependency? Timm now installs huggingface hub and has restrictions with older PyTorch versions. See: https://github.com/huggingface/pytorch-image-models/blob/main/requirements.txt